test/postMessageSpec.js

```diff
@@ -109,4 +162,56 @@ describe('POST /messages', function () {
       }
     })
   })
+
+  it('relays a message with "from"/"to"', function * () {
+    const message = this.fromToMessage
+    const listener = sinon.spy()
+    this.socket.on('message', (msg) => listener(JSON.parse(msg)))
+
+    yield timingHelper.sleep(50)
+    yield this.request()
+      .post('/messages')
+      .auth('alice', 'alice')
+      .send(message)
+      .expect(201)
+      .end()
+    yield timingHelper.sleep(50)
+
+    sinon.assert.calledTwice(listener)
+    sinon.assert.calledWith(listener.firstCall, { type: 'connect' })
+    sinon.assert.calledWith(listener.secondCall, {
+      type: 'message',
+      resource: {
+        ledger: 'http://localhost',
+        account: 'http://localhost/accounts/alice',
+        data: {foo: 'bar'}
+      }
+    })
+  })
+
+  it('relays a message when the admin is impersonating another user', function * () {
```

Comment:
It already does; see https://github.com/interledgerjs/five-bells-ledger/pull/348/files#diff-f4df84f46f3dcc8d94f3ede6fca6832eR124